### DISCUSSION TITLE
Same constraints' Jacobian but different initial feasibility

### DISCUSSION THREAD

**Discussion by giu67890** [casadi/casadi#3810] [Q&A]
Hi all,

two months ago I wrote a MATLAB script that tries to solve the time-optimal planning problem for redudant robots.
It gave me a good solution and I was pretty happy about it. This is the obtained constraints' jacobian and the first two iterations:
![Immagine 2024-08-16 123336](https://github.com/user-attachments/assets/a89c9543-943f-4ec8-a194-d258d8e9a6df)
Some days ago I did need this script to solve the problem with different trajctories but I found out that the script does not work as intended (and as it did two months ago). 
The jacobian structure is exactly the same, as well as the script's code, the initial guess and the solver, but this is how it appears now:
![Immagine 2024-08-16 123713](https://github.com/user-attachments/assets/6fa59074-08f0-4837-881e-3cdb5c69f6a9)
Do you have any ideas to help me?

Thanks in advance,
Giuseppe

**Comment by jgillis** [comment]
inf_pr for iteration zero gives a clue: the constraints evaluated for the initial guess are different for both.
Did you upgrade casadi version in the meantime?

**Reply by giu67890** [reply]
The things is: the code has been untouched since two months ago. It is really weird actually..
I did upgrade the version of casadi some time ago, and some days ago I had tried to restore a previous one, but nothing has changed.
